### PR TITLE
Feature add outcome to feedback

### PR DESCRIPTION
**Pull request by howardcochran**
Implements #77 "Improvements to ExePathFeedback
(and MoveBaseFeedback)"

### CODE REVIEW

File: mbf_abstract_nav/src/recovery_action.cpp [hunk 1]
```diff
@@ -102,8 +102,16 @@ void RecoveryAction::run(GoalHandle &goal_handle, AbstractRecoveryExecution &exe
 
       case AbstractRecoveryExecution::RECOVERY_DONE:
         recovery_active = false; // stopping the action
-        result.outcome = mbf_msgs::RecoveryResult::SUCCESS;
-        result.message = "Recovery \"" + goal.behavior + "\" done!";
+        result.outcome = execution.getOutcome();
+        result.message = execution.getMessage();
+        if (result.message.size() == 0)
```
Comment (spuetz) [inline mbf_abstract_nav/src/recovery_action.cpp]: if(result.message.empty())

File: mbf_abstract_nav/include/mbf_abstract_nav/controller_action.h [hunk 2]
```diff
@@ -66,6 +66,14 @@ class ControllerAction :
 
   void run(GoalHandle &goal_handle, AbstractControllerExecution &execution);
 
+protected:
+  void fillExePathFeedback(
```
Comment (spuetz) [inline mbf_abstract_nav/include/mbf_abstract_nav/controller_action.h]: Better call it `publishExePathFeedback()` and use the `goal_handle` as parameter.

**Review by spuetz — CHANGES_REQUESTED** [review]

**Comment by spuetz** [comment]
@howardcochran Would be cool if you do the changes I've suggested. After that we could merge that. :)

**Comment by howardcochran** [comment]
Thanks for the feedback, @spuetz.  I have rewritten the commits with the requested changes and rebased the branch on 0.2.1.

**Pull request merged**